components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.ts

```diff
@@ -33,27 +37,36 @@ export class ProjectsFilterDropdownComponent {
   // so they can be filtered while maintaining the actual options.
   filteredOptions: ProjectsFilterOption[] = [];
 
-  filterValue = '';
-
   optionsEdited = false;
 
   dropdownActive = false;
 
+
+  ngOnInit() {
+    this.projectsFilterService.filterValue$.subscribe((filterValue) => {
+      this.filteredOptions = this.filterOptions(filterValue);
+    });
+  }
+
   resetOptions() {
     if (!this.optionsEdited) {
       this.filteredOptions = this.editableOptions = cloneDeep(this.options);
     }
   }
 
-  handleFilterKeyUp(): void {
-    this.filteredOptions = this.filterOptions(this.filterValue);
+  handleFilterKeyUp(filterValue: string): void {
+    this.projectsFilterService.updateFilterValue(filterValue);
   }
 
   filterOptions(value: string): ProjectsFilterOption[] {
     return this.editableOptions.filter(option =>
       option.label.toLowerCase().indexOf(value.toLowerCase()) > -1);
   }
 
+  clearFilterValue(): void {
+    this.projectsFilterService.updateFilterValue('');
+  }
```

Comment:
Because this is only used once it seems like it would be better inlined... but that led me thinking about this.
1. Create this method:
```
  private setFilterValue(newValue: string): void {
    this.projectsFilterService.updateFilterValue(newValue);
  }
```
2. Use `setFilterValue` in both `handleFilterKeyUp` and `handleEscape`.

(Just a suggestion; you might prefer otherwise.)